insights/specs/default.py

```diff
@@ -63,6 +63,7 @@
     ls,
     lsattr,
     luks_devices,
+    nmcli,
```

Comment:
If we mean to arrange these imports in alphabet order, this `nmcli` shall be put after line `mount as mount_ds,`.